src/SWP/Bundle/CoreBundle/Theme/TenantAwareThemeContext.php

```diff
@@ -72,6 +72,9 @@ public function getTheme()
     {
         /* @var ThemeAwareTenantInterface $tenant */
         $tenant = $this->tenantContext->getTenant();
+        if (null === $tenant) {
```

Comment:
probably it wasn't called on cache clear. In `cache:clear` command we don't have tenant - we shouldn't have so this is ok.